toolkit/resources/imageconfigs/iso_initrd.json

```diff
@@ -31,6 +31,7 @@
                 "../assets/efi/grub/grub.cfg": "/installer/efi/grub/grub.cfg",
                 "../assets/efi/grub/grubEncrypt.cfg": "/installer/efi/grub/grubEncrypt.cfg",
                 "../assets/grub2/grub.cfg": "/installer/grub2/grub.cfg",
+                "../assets/grub2/grub": "/installer/grub2/grub",
```

Comment:
will go away once we rebase onto this fix already in main https://github.com/microsoft/CBL-Mariner/pull/6332/files